docker/run_elasticsearch

```diff
@@ -1 +1 @@
-docker run -d --net curation -p 9200:9200 -p 9300:9300 -e "xpack.security.enabled=false" -e "ingest.geoip.downloader.enabled=false" -e "discovery.type=single-node" -e "ES_JAVA_OPTS=-Xmx8g -Xms8g" --name elasticsearch docker.elastic.co/elasticsearch/elasticsearch:7.14.0
+docker run -d --net curation -p 9200:9200 -p 9300:9300 -e "xpack.security.enabled=false" -e "ingest.geoip.downloader.enabled=false" -e "discovery.type=single-node" -e "ES_JAVA_OPTS=-Xmx12g -Xms12g" --name elasticsearch docker.elastic.co/elasticsearch/elasticsearch:7.14.0
```

Comment:
Some devs do not have this much memory please revert back to 8.

Reply:
Ok. But people may run into memory issue the other way, ie at runtime.